mathics/doc/documentation/1-Manual.mdoc

```diff
@@ -36,30 +36,36 @@ Performance of \Mathics is not, right now, practical in large-scale projects and
 
 <section title="What does \Mathics offer?">
 
-Some of the features of \Mathics are:
+Some of the features of \Mathics tries to be compatible with Wolfram Alpha within the confines of the Python ecosystem.
```

Comment:
Why Wolfram alpha? I guess that Wolfram Language or if you want, Wolfram kernel would be a  more accurate description. wolfram alpha points more to a natural language web interface, I guess

Reply:
> Why Wolfram alpha?

Sorry, since I don't use this stuff, I often get this wrong. 

Is "Wolfram-Language kernel" okay?